lib/chromedriver.js

```diff
@@ -216,7 +216,9 @@ class Chromedriver extends events.EventEmitter {
     // on Android 7-9 webviews are backed by the main Chrome, not the system webview
     if (this.adb) {
       const apiLevel = await this.adb.getApiLevel();
-      if (apiLevel >= 24 && apiLevel <= 28) {
+      if (apiLevel >= 24 && apiLevel <= 28 &&
+          (this.bundleId === WEBVIEW_SHELL_BUNDLE_ID
```

Comment:
`[WEBVIEW_SHELL_BUNDLE_ID, ...WEBVIEW_BUNDLE_IDS].includes(this.bundleId)`

Reply:
Updated, please have a look.